package.json

```diff
@@ -66,6 +66,8 @@
   },
   "license": "MIT",
   "dependencies": {
```

Comment:
Probably not? I did to show we have no explicit dependencies...unsure what the best option is there.